src/store/wizardSlice.ts

```diff
@@ -835,10 +837,12 @@ export const wizardSlice = createSlice({
     },
     removePackage: (
       state,
-      action: PayloadAction<IBPackageWithRepositoryInfo['name']>,
+      action: PayloadAction<IBPackageWithRepositoryInfo>,
     ) => {
       const index = state.packages.findIndex(
-        (pkg) => pkg.name === action.payload,
+        (pkg) =>
```

Comment:
Similar here - there can't be two packages with the same in the state, so we don't need to differentiate between them by stream... all the names in the packages array should be unique, no? A side note - when you select a specific module, you should be able to select other package from the same module only, other module version and non-modular packages should have disabled checkboxes.